### ISSUE TITLE
Add fullname url

### ISSUE THREAD

**Issue by SergioGregorutti** [Viima/jquery-comments#18]
Hi,

It will be nice if we can add a new option on the comments array to add a link to the user profile page on the user name.

Makes sense?

Thanks in advance and thaks again for the plugin! It works like a charm.

**Comment by jjtykkyl** [comment]
This is actually pretty good idea and definitely worth implementing. The case is that I'm way too busy with other projects at the moment so I don't have the time to do this now. However, you are more than welcome to implement it yourself and create a pull-request out of it.

**Comment by SergioGregorutti** [comment]
No problem, this is just a suggestion. If I can help I'll do it as soon as I can.

Thanks!

**Comment by sadortun** [comment]
@SergioGregorutti are you going to make a PR with this ? It seems like a good idea !

As a side note, If you page is simply based on the ID, you can use a small piece of jQuery::
```javascript
$('.comment-wrapper > img, .comment-wrapper > div.name').on('click' , function (e)
{
     var url= '/some/page' + $(this).parent().parent().data('id')
     console.log(url); // location.href = url
});
```

**Comment by jjtykkyl** [comment]
I'm happy to let you know that this feature has been implemented to version 1.1.0
